### PR TITLE
Deployment Of Contracts on Matic Testnet Network

### LINKED ISSUE #146: Migration to Matic Mumbai Testnet Network
we need to Deploy the contracts on Matic Testnet, currently we are live on goerli

### PR DESCRIPTION
**Pull request by adi44**
Deployed Contracts on matic testnet as we will be soon migrating to matic

Fixes #146

### CODE REVIEW

**Review by SkandaBhat — APPROVED** [review]

**Review by abhishekvispute — DISMISSED** [review]

File: deployed/matic/abi/contracts/Core/interface/IJobManager.sol/IJobManager.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
+{
```
Comment (hrishikeshio) [inline deployed/matic/abi/contracts/Core/interface/IJobManager.sol/IJobManager.json:1]: do we need all these files? these should be in .gitignore as they are generated during deployment right?
This is cluttering the commit.

File: hardhat.config.js [hunk 2]
```diff
@@ -60,6 +61,13 @@ module.exports = {
       gas: 7700000,
       gasPrice: 1 * GWEI,
     },
+    matic: {
```
Comment (hrishikeshio) [inline hardhat.config.js:64]: Again matic what? testnet or mainnet? which testnet? needs to be explicitly mentioned.

File: package.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "coverage:check": "istanbul check-coverage --statements 97 --branches 85 --functions 97 --lines 97",
     "deploy:ganache": "./scripts/deploy.sh ganache",
     "deploy:goerli": "./scripts/deploy.sh goerli",
+    "deploy:matic": "./scripts/deploy.sh matic",
```
Comment (hrishikeshio) [inline package.json:17]: which matic network?

**Review by hrishikeshio — CHANGES_REQUESTED** [review]

File: hardhat.config.js (hunk 2 above)
Comment (SkandaBhat) [reply hardhat.config.js:64]: +1

File: package.json (hunk 3 above)
Comment (SkandaBhat) [reply package.json:17]: +1

File: deployed/matic/abi/contracts/Core/interface/IJobManager.sol/IJobManager.json (hunk 1 above)
Comment (SkandaBhat) [reply deployed/matic/abi/contracts/Core/interface/IJobManager.sol/IJobManager.json:1]: These are meant to be pushed for now. Replacing with npm deployment in a future commit.

Comment (SkandaBhat) [reply deployed/matic/abi/contracts/Core/interface/IJobManager.sol/IJobManager.json:1]: We commit these ABIs as part of deployment as of now - https://github.com/razor-network/contracts/blob/master/deployed/goerli/abi/contracts/Delegator.sol/Delegator.json

Comment (SkandaBhat) [reply deployed/matic/abi/contracts/Core/interface/IJobManager.sol/IJobManager.json:1]: Resolving this now @hrishikeshio since it is expected for now.

**Review by SkandaBhat — APPROVED** [review]

**Pull request merged**